cmd/resim/commands/batch.go

```diff
@@ -346,6 +351,93 @@ func createBatch(ccmd *cobra.Command, args []string) {
+	if err != nil {
+		log.Fatal("unable to retrieve system for batch:", err)
+	}
+	ValidateResponse(http.StatusOK, "unable to retrieve system for batch", systemResponse.HTTPResponse, systemResponse.Body)
+	system = systemResponse.JSON200
+
+	// Intro text
+	introData := struct {
+		SuiteUrl   string
+		SuiteName  string
+		BatchUrl   string
+		SystemUrl  string
+		SystemName string
+	}{
+		fmt.Sprintf("%s/test-suites/%s/revisions/%d", baseUrl, batch.TestSuiteID.String(), *batch.TestSuiteRevision),
+		suite.Name,
+		fmt.Sprintf("%s/batches/%s", baseUrl, batch.BatchID.String()),
+		fmt.Sprintf("%s/systems/%s", baseUrl, batch.SystemID.String()),
+		system.Name,
+	}
+	introTemplate := template.Must(template.New("intro").Parse("Last night’s <{{.SuiteUrl}}|{{.SuiteName}}> *<{{.BatchUrl}}|run>* for <{{.SystemUrl}}|{{.SystemName}}> ran successfully with the following breakdown:"))
+	var introBuffer bytes.Buffer
+	introTemplate.Execute(&introBuffer, introData)
+	introTextBlock := slack.NewTextBlockObject("mrkdwn", introBuffer.String(), false, false)
+	blocks.BlockSet[0] = slack.NewSectionBlock(introTextBlock, nil, nil)
+
+	// List section
+	boldStyle := slack.RichTextSectionTextStyle{Bold: true}
+	buildListElement := func(count int, label string, filter string) *slack.RichTextSection {
+		return slack.NewRichTextSection(
+			slack.NewRichTextSectionTextElement(fmt.Sprintf("%d ", count), nil),
+			slack.NewRichTextSectionLinkElement(introData.BatchUrl+"?performanceFilter="+filter, label, &boldStyle),
+		)
+	}
+
+	listBlock := slack.NewRichTextList("bullet", 0,
+		slack.NewRichTextSection(slack.NewRichTextSectionTextElement(fmt.Sprintf("%d total tests", *batch.TotalJobs), nil)),
+		// Passed calculation borrowed from bff: https://github.com/resim-ai/rerun/blob/ebf0cde9472f555ae099e08e512ed4a7dfdf01f4/bff/lib/bff/batches/conflated_status_counts.ex#L49
+		buildListElement(batch.JobStatusCounts.Succeeded-(batch.JobMetricsStatusCounts.FailBlock+batch.JobMetricsStatusCounts.FailWarn), "Passed", "Passed"),
+		buildListElement(batch.JobMetricsStatusCounts.FailBlock, "Blocking", "Blocker"),
+		buildListElement(batch.JobMetricsStatusCounts.FailWarn, "Warning", "Warning"),
+		buildListElement(batch.JobStatusCounts.Error, "Erroring", "Error"),
+	)
+	blocks.BlockSet[1] = slack.NewRichTextBlock("list", listBlock)
+
+	// Report section
+	reportResponse, err := Client.ListReportsWithResponse(context.Background(), *batch.ProjectID, &api.ListReportsParams{Search: Ptr(fmt.Sprintf("test_suite_id = \"%s\"", batch.TestSuiteID.String())), OrderBy: Ptr("timestamp")})
+	if err != nil {
+		log.Fatal("unable to retrieve report for batch:", err)
+	}
+	ValidateResponse(http.StatusOK, "unable to retrieve report for batch", reportResponse.HTTPResponse, reportResponse.Body)
+	reports = reportResponse.JSON200.Reports
+	if len(*reports) > 0 {
+		reportFooter := fmt.Sprintf("See more historical details in the nightly *<%s/reports/%s|Report>*.", baseUrl, (*reports)[0].ReportID)
```

Comment:
```suggestion
		reportFooter := fmt.Sprintf("See more historical details in the latest *<%s/reports/%s|Report>*.", baseUrl, (*reports)[0].ReportID)
```

(I wonder if we should actively give the date of the last report?) it could technically be super old

Reply:
Maybe we should just look at when the report was last updated and if it's more than 48 hours we ignore it?

Reply:
I forgot that reports are a separate step from the batch. I'll revisit this section later - there's really no nice way to wait for the report to be done on the batch level that I can think of.